go/api/glide_cluster_client.go

```diff
@@ -69,3 +74,28 @@ func (client *GlideClusterClient) CustomCommand(args []string) (ClusterValue[int
 	}
 	return CreateClusterValue(data), nil
 }
+
+// Returns the number of keys in the currently selected database.
```

Comment:
Cluster installation has only 1 database (dislike the stadnalone)
```suggestion
// Returns the number of keys in the database.
```